libcontainer/specconv/spec_linux.go

```diff
@@ -333,7 +333,10 @@ func CreateLibcontainerConfig(opts *CreateOpts) (*configs.Config, error) {
 
 func createLibcontainerMount(cwd string, m specs.Mount) (*configs.Mount, error) {
 	if !filepath.IsAbs(m.Destination) {
-		return nil, fmt.Errorf("mount destination %s not absolute", m.Destination)
+		// Relax validation for backward compatibility
+		// TODO (runc 1.2.0): replace a warning below with
+		// return nil, fmt.Errorf("mount destination %s not absolute", m.Destination)
+		logrus.Warnf("mount destination %s not absolute (this won't be supported past runc 1.2.0)", m.Destination)
```

Comment:
We are still not sure how our post-v1 version scheme will look like.  
So "late 2021" might be better than "1.2.0"?

Reply:
Think we should make it more "generic"?

```suggestion
		logrus.Warnf("mount destination %s not absolute (this will no longer be supported in a future release)", m.Destination)
```

Reply:
Updated the description and TODO